docs/resources/Channel.md

```diff
@@ -525,21 +525,22 @@ Get a channel by ID. Returns a [channel](#DOCS_RESOURCES_CHANNEL/channel-object)
-| permission_overwrites | array of [overwrite](#DOCS_RESOURCES_CHANNEL/overwrite-object) objects | channel or category-specific permissions                                                                                                                                        | All          |
-| parent_id             | snowflake                                                              | id of the new parent category for a channel                                                                                                                                     | Text, Voice  |
+| Field                 | Type                                                                   | Description                                                                                                                                                                     | Channel Type              |
+| --------------------- | ---------------------------------------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ------------------------- |
+| name                  | string                                                                 | 2-100 character channel name                                                                                                                                                    | All                       |
+| type                  | integer                                                                | the [type of channel](#DOCS_RESOURCES_CHANNEL/channel-object-channel-types); only conversion between text and news is supported and only in guilds with the "NEWS" feature      | Text, News                |
+| position              | integer                                                                | the position of the channel in the left-hand listing                                                                                                                            | All                       |
+| topic                 | string                                                                 | 0-1024 character channel topic                                                                                                                                                  | Text, News                |
+| nsfw                  | boolean                                                                | whether the channel is nsfw                                                                                                                                                     | Text, News, Store         |
```

Comment:
Store channels cannot be marked as NSFW

Reply:
How did you come to this conclusion?

Reply:
There's no NSFW toggle in the channel settings. However, after further testing, looks like you can set nsfw in the API and Discord stores it, but it's not shown in the client, so that's probably a bug,